src/libraries/System.Diagnostics.EventLog/src/System/Diagnostics/Reader/EventLogException.cs

```diff
@@ -16,30 +17,30 @@ internal static void Throw(int errorCode)
         {
             switch (errorCode)
             {
-                case 2:
-                case 3:
-                case 15007:
-                case 15027:
-                case 15028:
-                case 15002:
+                case 2: // ERROR_FILE_NOT_FOUND
```

Comment:
I don't know why I suggested comments. I should have suggested named constants, ie Interop.Errors.ERROR_FILE_NOT_FOUND etc. My bad. Your change above reminded me.